Java/crypto/src/main/java/com/breadwallet/crypto/TransferConfirmation.java

```diff
@@ -21,12 +21,16 @@
     private final UnsignedLong transactionIndex;
     private final Date timestamp;
     private final Optional<Amount> fee;
+    private final Boolean success;
```

Comment:
Use `boolean` here, no?

Reply:
I changed it to `boolean` but the use of `UnsignedLong` lead me to use the class-based `Boolean` in the first place.

Reply:
Yeah, the UnsignedLong is because Java doesn't have that type natively.